lib/runner/cli/clirunner.js

```diff
@@ -620,6 +651,171 @@ CliRunner.prototype = {
     });
   },
 
+  startTestWorkers : function(finishCallback) {
```

Comment:
This method is a bit too big and seems to duplicate the walk.js code, which probably wasn't easy to reuse at the time of writing this but it might be now. Also it can probably share some code with startChildProcesses which would also help making the later more readable.